x-pack/plugins/osquery/server/plugin.ts

```diff
@@ -21,6 +27,168 @@ import { initSavedObjects } from './saved_objects';
 import { initUsageCollectors } from './usage';
 import { OsqueryAppContext, OsqueryAppContextService } from './lib/osquery_app_context_services';
 import { ConfigType } from './config';
+import { packSavedObjectType, savedQuerySavedObjectType } from '../common/types';
+import { PLUGIN_ID } from '../common';
+
+const registerFeatures = (features: SetupPlugins['features']) => {
+  features.registerKibanaFeature({
```

Comment:
I thought you were going to use `order: 2300` here, did it get removed on accident or on purpose? 
Just wanted to double check :smile: